src/utils/availability.js

```diff
@@ -0,0 +1,8 @@
+
+let availabilityChannel = undefined;
+const getAvailabilityChannel = () => {return availabilityChannel}
```

Comment:
Suggest we make this an async function. It doesn't need it right now, but I would anticipate that in the near future we'll need to make a db call to get this information, which will be async. So make it async now (just throw the async keyword on the front) for less refactoring later.